PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/nucleus/NucleusData.java

```diff
@@ -0,0 +1,311 @@
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Reason", Icon.called("comment").of(Color.YELLOW).of(Family.REGULAR)), reason);
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Jail", Icon.called("bars").of(Color.YELLOW).of(Family.SOLID)), inmate.getJailName());
+            }
+        }
+
+        if (NucleusAPI.getHomeService().isPresent()) {
+            NucleusHomeService homeService = NucleusAPI.getHomeService().get();
+
+            int homeCount = homeService.getHomeCount(user);
+            int maxHomes = homeService.getMaximumHomes(user);
+
+            inspectContainer.addValue("&nbsp;" + getWithIcon("Homes", Icon.called("home").of(Color.GREEN).of(Family.SOLID)), homeCount + "/" + maxHomes);
+
+            List<Home> homes = homeService.getHomes(user);
+
+            if (!homes.isEmpty()) {
+                TableContainer homesTable = new TableContainer(getWithIcon("Home", Icon.called("home").of(Family.SOLID)));
+                homesTable.setColor("light-green");
+
+                for (Home home : homes) {
+                    homesTable.addRow(home.getName());
+                }
+
+                inspectContainer.addTable("Homes", homesTable);
+            }
+        }
+
+        if (NucleusAPI.getInvulnerabilityService().isPresent()) {
+            boolean invulnerable = NucleusAPI.getInvulnerabilityService().get().isInvulnerable(user);
+
+            inspectContainer.addValue(getWithIcon("Invulnerable", Icon.called("crosshairs").of(Color.BLUE).of(Family.SOLID)), invulnerable ? "Yes" : "No");
+        }
+
+        if (NucleusAPI.getNicknameService().isPresent()) {
+            Optional<Text> nickname = NucleusAPI.getNicknameService().get().getNickname(user);
+
+            if (nickname.isPresent()) {
+                String nicknameString = HtmlUtils.swapColorsToSpan(TextSerializers.FORMATTING_CODE.serialize(nickname.get()));
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Nickname", Icon.called("id-badge").of(Color.GREEN).of(Family.REGULAR)), nicknameString);
+            }
+        }
+
+        if (NucleusAPI.getNoteService().isPresent()) {
+            NucleusNoteService noteService = NucleusAPI.getNoteService().get();
+
+            List<Note> notes = noteService.getNotes(user);
+            if (!notes.isEmpty()) {
+                TableContainer notesTable = new TableContainer(
+                        getWithIcon("Noter", Icon.called("pen").of(Family.SOLID)),
+                        getWithIcon("Note", Icon.called("sticky-note").of(Family.REGULAR))
+                );
+
+                notesTable.setColor("light-blue");
+
+                for (Note note : notes) {
+                    String noter = "Unknown";
+                    if (note.getNoter().isPresent()) {
+                        User noterUser = getUser(note.getNoter().get());
```

Comment:
Can be simplified with Optional#map
`note.getNoter().map(this::getUser).orElse(null)`